ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -53,6 +53,7 @@ let initializeInfo : InitializeResult.t =
 let ocamlmerlin_reason = "ocamlmerlin-reason"
 
 let send_diagnostics rpc doc =
+  let diagnostic_create = Diagnostic.create ~source:"ocaml-lsp" in
```

Comment:
how about `ocamllsp`? That's the name of the binary at least.

Reply:
Fine by me. I wasn't sure which name to use.